great_expectations/data_context/store/expectations_store.py

```diff
@@ -97,7 +99,7 @@ class ExpectationsStore(Store):
 
     _key_class = ExpectationSuiteIdentifier
 
-    def __init__(self, store_backend=None, runtime_environment=None):
+    def __init__(self, store_backend: dict = None, runtime_environment: dict = None):
```

Comment:
nit, but I believe the preferred typing solution here is `Dict` vs `dict` so that you could add in the key/value types. Not worth changing, but FYI.

Reply:
Good call - I just checked though and since python 3.9 using `Dict` is deprecated since `builtins.dict` supports the `[]`: `https://docs.python.org/3/library/typing.html#typing.Dict`. However: "To annotate arguments it is preferred to use an abstract collection type such as Mapping." Will leave as is for now until I understand better how best to use this.